app/models/friendship.rb

```diff
@@ -3,4 +3,7 @@
 class Friendship < ApplicationRecord
   belongs_to :user
   belongs_to :friend, class_name: 'User'
+  validates :user, presence: true
+  validates :friend, presence: true
```

Comment:
belongs to automatically validates the model you don't need use these.

Reply:
Thanks for the suggesting @ispirett. We'll remove it for our next milestone.